frontend/src/core/websocket/useWebSocket.tsx

```diff
@@ -35,6 +35,9 @@ export function useWebSocket(options: UseWebSocketOptions) {
             maxRetries: 10,
             debug: false,
             startClosed: true,
+            // long timeout -- the server can become slow when many notebooks
+            // are open.
+            connectionTimeout: 60_000,
```

Comment:
looks like the default is 4s (just to have it handy here) https://github.com/partykit/partykit/blob/50715a06242732ac828993d2c15d63d30846efb2/packages/partysocket/src/ws.ts#L120